### DISCUSSION TITLE
Hello, I cannot apply for the github tutorial because the camera is not working.

### DISCUSSION THREAD

**Discussion by EkiciFurkan** [community/community#140217] [GitHub Education]
![WhatsApp Görsel 2024-09-30 saat 22 20 58_89d3d5d3](https://github.com/user-attachments/assets/9f0113cb-c974-4dc2-8669-901fe1a3adb8)

What should I do in this situation

**Comment by 7x9z** [comment]
Hello @EkiciFurkan,
For this to work you will need to enable or allow GitHub to use your phone's camera in the app settings or the site's settings.

**Reply by EkiciFurkan** [reply]
It seems to have all the necessary permissions, and the front camera works well, but the back camera doesn’t work on any phone.

**Reply by 7x9z** [reply]
Hello @EkiciFurkan,
This seems to be a issue with the app I think, If you can use the front camera to try and verify.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @EkiciFurkan ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by EkiciFurkan** [reply]
Thanks for information !
